Pd 304 doc for cloud snap enumerate api change (#496)
portworx/pxdocs: content/reference/CLI/cloud-snaps.md

 Once the volume is backed up to the cloud successfully, listing the remote cloudsnaps will display the backup that has just been completed.

### List your cloud backups

> Use `pxctl cloudsnap list` command to list your cloud backups:

Reviewer: ```suggestion
Use the `pxctl cloudsnap list` command to list your cloud backups:
```

You can restore a backup of a _PX_ to one of your _PX_ volumes in the cluster. Once restored, the volume inherits the attributes from the backup (e.g.: file system, size and block size). The replication level of the restored volume defaults to 1, irrespective of the replication level of the volume that was backed up. Users can increase the replication factor once the restore is complete on the restored volume.

To restore a backup from cloud, enter the following command:

```text
pxctl cloudsnap restore --snap 2e4d4b67-95d7-481e-aec5-14223ac55170/56706279008755778-725134927222077463
```

```output
Cloudsnap restore started successfully on volume: 104172750626071399 with task name:59c4cfd5-4160-45db-b326-f37b327d9225
```

Once a restore gets started, `pxctl` shows the id of the volume created to restore the cloud snap together with the task-id.

> While the restore process is running, run the `pxctl cloudsnap status` to see its status:

Reviewer: ```suggestion
While the restore process is running, run the `pxctl cloudsnap status` command to see its status:
```

```text
pxctl cloudsnap status
```

```output
NAME                    SOURCEVOLUME                                    STATE        NODE        BYTES-PROCESSED    TIME-ELAPSED    COMPLETED
3f4f0a67-e12a-4d35-81ad-985657757352    2e4d4b67-95d7-481e-aec5-14223ac55170/56706279008755778-725134927222077463    Backup-Done    70.0.73.246    11988570112    3m29.825766964s    Thu, 17 Jan 2019 00:07:29 UTC
39f66859-14b1-4ce0-a4c0-c858e714689e    2e4d4b67-95d7-481e-aec5-14223ac55170/590114184663672482-951325819047337066-incr    Backup-Done    70.0.73.246    420044800    17.460186585s    Wed, 16 Jan 2019 22:27:30 UTC
59c4cfd5-4160-45db-b326-f37b327d9225    2e4d4b67-95d7-481e-aec5-14223ac55170/212160250617983239-283838486341798860    Restore-Done    70.0.73.246    1079287808    3.174541219s    Thu, 17 Jan 2019 00:15:19 UTC
```

> If you want to see the status of a particular process, run the `pxctl cloudsnap status` and pass it the `--name` flag with the name of the task you want to inspect:

Reviewer: ```suggestion
If you want to see the status of a particular process, run the `pxctl cloudsnap status` command and pass it the `--name` flag with the name of the task you want to inspect:
```

```text
pxctl cloudsnap status --name 59c4cfd5-4160-45db-b326-f37b327d9225
```

```output
59c4cfd5-4160-45db-b326-f37b327d9225    2e4d4b67-95d7-481e-aec5-14223ac55170/212160250617983239-283838486341798860    Restore-Done    70.0.73.246    1079287808    3.174541219s    Thu, 17 Jan 2019 00:15:19 UTC
```

If the restore command fails, it shows the reason why it failed.

Note that the restored volume will not be attached or mounted automatically.

{{<info>}}
{{% content "reference/CLI/shared/optimized-restores-definition.md" %}}

> For more details, check out [this link](/reference/cli/cluster/#enabling-optimized-restores)

Reviewer: A better practice when placing links in documentation is to name the section rather than just saying "this link" or "click here". This helps users understand where they're going and also helps them find an article if the link ends up broken. 

If you want to take it one step further, briefly describe what the "more details" are. 

```suggestion
For more details about optimized restores, visit the [Enabling optimized restores](/reference/cli/cluster/#enabling-optimized-restores) section. 
```
